clark_trot_controller/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(clark_trot_controller)
```

Comment:
Let's not call this clark trot, since it isn't really his idea. Maybe just open_loop_trot or something actually descriptive.

Reply:
Sure, will modify and update PR